akka-operator/values.yaml

```diff
@@ -43,3 +43,8 @@ provider:
 gcp:
   repository: gcr.io/cloud-marketplace/lightbend-public/akka-cloud-platform
   deployer: false
+# Commercial image credentials - set `username` and `password` to install as Kubernetes Secret.
+imageCredentials:
```

Comment:
the "helm idiomatic" empty object would be nice here:
```yaml
imageCredentials: {}
```

Reply:
Using the alternative I mentioned above, this is no longer an empty object since the repository always exists.